keps/sig-node/2570-memory-qos/README.md

```diff
@@ -0,0 +1,302 @@
+We need add new field `Unified` in CRI api which is basically passthrough for OCI spec Unified field and has same semantics: opencontainers/runtime-spec#1040
+
+```
+type LinuxContainerResources struct {
+    ...
+    Unified map[string]string `json:"unified,omitempty"
+}
+```
+
+### Test Plan
+For `Alpha`, unit tests were added to test functionality for container/pod/node level cgroup with containerd and CRI-O.
+
+For `Beta`, e2e-node tests were added, tests were added for dockershim
+
+For `GA`, the introduced e2e tests will be promoted to conformance. It was also verified that all e2e coverage was proper and CRI's had tests in their respective repos testing this feature.
+
+
+### Graduation Criteria
+
+#### Alpha Graduation
+- [cgroup_v2](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2254-cgroup-v2) is in `Alpha`
+- Memory QoS is implemented for new feature gate
+- Memory QoS is covered by proper tests
+- Memory QoS supports containerd, cri-o
+
+#### Beta Graduation
+- [cgroup_v2](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2254-cgroup-v2) is in `Beta`
+- Memory QoS is covered by unit and e2e-node tests
+- Memory QoS supports containerd, cri-o and dockershim
+
+#### GA Graduation
+- [cgroup_v2](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2254-cgroup-v2) is in `GA`
+- Memory QoS has been in beta for at least 2 releases
+- Memory QoS sees use in 3 projects or articles
+- Memory QoS is covered by conformance tests
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: MemoryQoS
+  - Components depending on the feature gate: kubelet
+- [x] Other
```

Comment:
What other mechanism is used? It's not just the feature gate?

Reply:
It is typos, have updated.